common/database_schema.h

```diff
@@ -232,6 +232,7 @@ namespace DatabaseSchema {
 			"spawnentry",
 			"spawngroup",
 			"spells_new",
+			"splash_faction",
```

Comment:
Maybe this should be called something like `faction_sets` or `faction_set_ratios`? At the very least `faction` as the prefix for the name I think would be better

Reply:
splash faction is the name I've seen people use for this concept so I went with splash faction. It was already established so I don't think there is a good reason not to use it.

Reply:
i vote for faction associations

Reply:
I agree, at least `faction_` something